UKHomeOffice/sgmr-service
# 2.1.0 - SPCR-331 Change nationality drop down to autocomplete component (Accessibility)

## Description
The Issue:  User is told to use the arrow keys to scroll down to the nationality of the pleasure craft. However, the arrow keys do not allow the user to get to a nationality after 'C' (it goes back up to B). If the user uses the keyboard to select a letter they are then able to use the arrow keys to scroll and select the correct nationality beginning with that letter.

The user should be informed that they have the option to do this.

Solution:  Change this to the new auto-complete input type

**The PleasureCraftForm test will fail, and will be fixed in the SPCR-256 branch once this is merged into it as the errors coming from that test do not pertain to this ticket specifically but that branch as a whole. All other tests should pass**

## To Test
- Reinstall packages
- Go to the Pleasure Crafts page
- Save a new pleasure craft
- Go to the second page of the form
- Click 'Yes' when asked 'Does this pleasure craft have a registration number?'
- Type 3 or more letters in the country of registration autocomplete box
- You should be presented with suggestions
- You should be able to use the arrow keys to cycle through the suggestions

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
